src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1613,13 +1613,21 @@ subroutine li_apply_front_ablation_velocity(meshPool, geometryPool, velocityPool
                .and. bedTopography(iCell) < config_sea_level &
                .and. li_mask_is_dynamic_ice(cellMask(iCell)) ) then
 
-               groundedMarineMarginMask(iCell) = 1
-            else
-               groundedMarineMarginMask(iCell) = 0
+               frontAblationMask(iCell) = 1
+               ! Include adjacent non-dynamic cells
```

Comment:
Can you explain why adjacent non-dynamic cells need to be included here and in the applyToFloating logic?  Also, if they do need to be included, do we need multiple passes of finding them to include those 'horn' locations of non-dynamic cells that don't have dynamic neighbors?